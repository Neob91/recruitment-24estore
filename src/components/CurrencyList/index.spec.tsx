import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { CurrencyList } from '.';

describe('CurrencyList', () => {
  it('renders correctly', () => {
    const r = new ShallowRenderer();

    r.render(<CurrencyList codes={['EUR', 'USD', 'AUD']} />);
    expect(r.getRenderOutput()).toMatchSnapshot();
  });
});
