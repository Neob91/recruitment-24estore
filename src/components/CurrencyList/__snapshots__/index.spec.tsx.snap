// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurrencyList renders correctly 1`] = `
<div>
  <Memo(Connect(Component))
    code="EUR"
  />
  <Memo(Connect(Component))
    code="USD"
  />
  <Memo(Connect(Component))
    code="AUD"
  />
</div>
`;
